packages/dds/tree/package.json

```
@@ -87,6 +87,7 @@
 		"test:benchmark:report": "mocha --exit --perfMode --parentProcess --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
 		"test:coverage": "c8 npm test",
 		"test:memory": "mocha --config ./src/test/memory/.mocharc.cjs",
+		"test:custom:shared-tree": "mocha --config ./.mochaRunCustomData.cjs",
```

(And if we go with my suggested filename for the mocha config, we'll have to update it here too)
```suggestion
		"test:customBenchmarks": "mocha --config ./.mochaRunCustomData.cjs",
```